TLM/TLM/State/Options.cs

```diff
@@ -33,6 +34,18 @@ public enum PersistTo {
             GlobalOrSavegame = Global | Savegame,
         }
 
+        /// <summary>
+        /// When <c>true</c>, options are safe to query.
+        ///
```

Comment:
I'd prefer it without the additional whitespace but ok ;)
This is also fine.

Reply:
Removed that method and it's xmldoc.